config/cli/common/main/packages.additional

```diff
@@ -47,7 +47,7 @@ qrencode
 rfkill
 rng-tools
 screen
-software-properties-common
+#software-properties-common
```

Comment:
Should be kept as Bookworm has this package and is _stable_. If this is an issue for _Trixie_ a separate package list needs to be added for it.

Reply:
Thank you @EvilOlaf for the review.  I certainly agree with you.  But that is the situation already as proposed with this review.

```
$ grep software-properties-common config/desktop/bookworm/environments/mate/config_base/packages
software-properties-common
```